models/DataObject/AbstractObject.php

```diff
@@ -1359,7 +1359,7 @@ public static function __callStatic($method, $arguments)
         $db = \Pimcore\Db::get();
 
         if (in_array(strtolower($realPropertyName), self::$objectColumns)) {
-            $value = $arguments[0] ?? null;
+            $value = empty($arguments[0]) ? throw new \InvalidArgumentException('Mandatory argument $value not set.') : $arguments[0];
```

Comment:
null, '', 0 should be possible. Use `!isset()`

Reply:
I'd write it like this, then: 
```suggestion
            $value = array_key_exists($arguments[0]) ? $arguments[0] : throw new \InvalidArgumentException('Mandatory argument $value not set.');
```

Reply:
I agree on using array_key_exists, things are a bit tricky here, because passing `''` or `null` doesn´t make a difference for strings, it will result in the same sql query. And with passing `''` you won´t find any newly created objects with textfields that haven´t been edited- because the corresponding database field is set to `NULL` and not `''`. I suspect this to be a defect and created an issue here: https://github.com/pimcore/pimcore/issues/13089 

Thanks for the help guys.